### ISSUE TITLE
Difference in trends for 7-day incidence and 7-day average

### ISSUE THREAD

**Issue by nilsalex** [corona-warn-app/cwa-app-ios#2002]
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Technical details

- Device name: iPhone 11
- iOS Version: 14.4 (18D52)
- App Version: 1.12.1 (0)

## Describe the bug

As of now (16.02.2021, 17:11 CET), CWA shows a 7-day average of 7,274 confirmed infections and a 7-day incidence of 58.7/100,000. For the 7-day average, an arrow pointing towards the lower right indicates a downward trend, while for the 7-day incidence, an arrow pointing to the right indicates a stable trend. Yesterday, the difference was even higher: a downward trend vs an upward trend.

My understand is, that both numbers are related by a factor like

```
(7-day incidence) = (7-day average) * 7 * 100,000 / (about 83,000,000)
```

and therefore, the trend should always be the same.

## Steps to reproduce the issue

Open the app and swipe through the widgets.

## Expected behaviour

Same trend for both indicators.

**Comment by Ein-Tim** [comment]
@nilsalex 

I think the cause for this is the following:

>Die Anzahl der Fälle - und deren Differenz zum Vortag - und die Anzahl der Todesfälle beziehen sich auf Fälle, die dem RKI täglich übermittelt werden. Dies beinhaltet Fälle, die am gleichen Tag oder bereits an früheren Tagen an das Gesundheitsamt gemeldet worden sind. Bei den Fällen in den letzten 7 Tagen und der 7-Tage-Inzidenz liegt das Meldedatum beim Gesundheitsamt zugrunde, also das Datum, an dem das lokale Gesundheitsamt Kenntnis über den Fall erlangt und ihn elektronisch erfasst hat.

(Source: https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Fallzahlen.html).

This would explain the difference, or?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
I'm definitely not an expert on these statistics, but I can Google!

Start first by tapping the ℹ️ icon in the app for the definitions.

Then access the raw data through
https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Fallzahlen.html and a link in that page to
https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Daten/Fallzahlen_Daten.html

According to that Excel file in tab "BL_7-Tage-Inzidenz" the 7-Day Incidence on Feb 16, 2021 of confirmed new infections was **58.7** and 7 days before that on Feb 9, 2021 it was 72.8. So that was a downwards trend of 14.1 or -19% based on the Feb 9 data.

Using the tab "BL_7-Tage-Fallzahlen" I couldn't find values which matched the ones in the app, so I used the tab "Fälle-Todesfälle-gesamt" instead.
The sum of Differenz Vortag Fälle for Feb 10 to Feb 16, 2021 is 50919, divided by 7 is **7274**.
The sum for Feb 3 to Feb 9, 2021 is 63839, divided by 7 is 9120.
This is a difference of 1846 or -20% compared to the Feb 9 data.

----

Based on that I don't understand why the app is showing Trend: Steady for the 7-Day Incidence when, according to the figure I quoted, the trend is 19% down and this is more than the 5% threshold to declare it as Trend: Downwards and mark it with a green arrow. 

**This needs to be looked at.**

Thanks to @nilsalex for bringing this up!

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Yes, this also affects Android, so it should be in the documentation repo.

I think it should be looked at urgently because the 7-Day Incidence value and the trend is the one figure that everybody, including politicians, are looking at to influence the decision about the easing of lockdown.

**Comment by MikeMcC399** [comment]
For ease of reference here are the RKI daily reports for Feb 16, 2021 and for 7 days previously on Feb 9, 2021.

[2021-02-09-en.pdf](https://github.com/corona-warn-app/cwa-app-ios/files/5990664/2021-02-09-en.pdf)
[2021-02-16-en.pdf](https://github.com/corona-warn-app/cwa-app-ios/files/5990667/2021-02-16-en.pdf)

These show the figures 

Date | 7-Day Incidence per 100,000 population
--- | ---
Feb 9, 2021 | 73
Feb 16, 2021 | 59

which is a clear downwards trend (that I am sure we are all happy to be seeing 👏!)

**Comment by Ein-Tim** [comment]
@dsarkar Since this seems urgent pinging you.

**Comment by MikeMcC399** [comment]
The value today, Feb 17, 2021, for 7-Day Incidence is 57.0 and the trend is down, which looks good.

The data for yesterday should still be investigated though.

**Comment by Ein-Tim** [comment]
@dsarkar Could you take a look at this and transfer it to the correct repo? 

Thanks!
